src/ngraph/frontend/onnx_import/utils/reshape.cpp

```diff
@@ -100,6 +99,15 @@ namespace ngraph
                              "Scalar value can't be derived from a node with ",
                              node_shape);
 
+                // If node is a Constant, recreate as Constant with Shape{}
+                if (node->is_constant())
+                {
+                    const auto value =
+                        ngraph::as_type_ptr<ngraph::op::Constant>(node)->get_data_ptr();
+                    return std::make_shared<ngraph::op::Constant>(
```

Comment:
I think this should be unit tested to verify that values in the reshaped constant propagate/broadcast correctly.

Reply:
Done.